libraries/stumbler/src/main/java/org/mozilla/mozstumbler/service/stumblerthread/scanners/WifiScanner.java

```diff
@@ -73,6 +74,8 @@ private boolean isScanEnabled() {
     }
 
     public synchronized void start(final ActiveOrPassiveStumbling stumblingMode) {
+        mPassiveScanCount.set(0);
```

Comment:
Add a comment:
// If the scan timer is active, this will reset the number of times it has run